opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -66,7 +67,8 @@ namespace Opm {
             AllProperties = IntProperties | DoubleProperties
         };
 
-        EclipseState(std::shared_ptr< const Deck > deck , const ParseContext& parseContext);
+        EclipseState(std::shared_ptr< const Deck > );
```

Comment:
> Since this constructor is new, can we use `Deck&` over `std::shared_ptr< Deck >`?

don't forget to mark it `explicit`. (that's also a good idea in its current form.)